src/main/com/fulcrologic/guardrails/config.cljc

```diff
@@ -85,12 +85,12 @@
                         (::value @*config-cache)
                         (::value (reset! *config-cache {::timestamp now
                                                         ::value     (reload-config)})))))]
-       #?(:clj (when (and result cljs.env/*compiler*)
```

Comment:
hm...I don't understand.  I tested this on a project and it worked.  I see you're indicating that the var is the wrong one, but I'm not sure how it ever worked in that case.

Reply:
Ah, I see...nevermind.